src/components/FormatNumber.js

```diff
@@ -0,0 +1,19 @@
+const numberFormater = (num) => {
+  let value;
+  if (Math.abs(num) > 999999999) {
+    value = `${Math.sign(num) * ((Math.abs(num) / 1000000000).toFixed(2))}b`;
+  } else if (Math.abs(num) > 999999) {
+    value = `${Math.sign(num) * ((Math.abs(num) / 1000000).toFixed(2))}m`;
+  } else if (Math.abs(num) > 999) {
+    value = `${Math.sign(num) * ((Math.abs(num) / 1000).toFixed(2))}k`;
+  } else {
+    value = Math.sign(num) * Math.abs(num);
+  }
+  return value;
+};
+
+export default numberFormater;
+// function kFormatter(num) {
+//   return Math.abs(num) > 999 ? Math.sign(num)*((Math.abs(num)/1000).toFixed(1))
+// + 'k' : Math.sign(num)*Math.abs(num)
+// }
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.